test/unit/tracks/text-track-display.test.js

```diff
@@ -183,6 +183,50 @@ if (!Html5.supportsNativeTextTracks()) {
     player.dispose();
   });
 
+  QUnit.test("don't select user langauge if it is an empty string", function(assert) {
+    const player = TestHelpers.makePlayer();
+    const track1 = {
+      kind: 'captions',
+      label: 'English',
+      language: 'en',
+      src: 'en.vtt'
+    };
+    const track2 = {
+      kind: 'captions',
+      label: 'Spanish',
+      language: 'es',
+      src: 'es.vtt'
+    };
+    const track3 = {
+      kind: 'metadata',
+      label: 'segment-metadata'
+    };
+
+    player.src({type: 'video/mp4', src: 'http://google.com'});
+    // manualCleanUp = true by default
+    const englishTrack = player.addRemoteTextTrack(track1, true).track;
```

Comment:
might want to add the metadata track first just to be extra sure we're fixing the issue

Reply:
it shouldn't matter the order they are added, especially since we're waiting till loadstart (or the tick) to do the work. Plus, we can see it fail and then succeed in the builds.